dags/glam_fenix.py

```diff
@@ -128,13 +128,38 @@
         )
         latest_versions = query(task_name=f"{product}__latest_versions_v1")
 
+        scalar_wait_for_yesterdays_aggregates = ExternalTaskSensor(
```

Comment:
What's the reason for doing it this way instead of `depends_on_past=True` with `max_active_runs=1`?